src/Conversion/ONNXToTOSA/Math/Elementwise.cpp

```diff
@@ -42,6 +42,38 @@ class ONNXElementwiseUnaryOpLoweringToTOSA
   }
 };
 
+template <typename ONNXOpT, typename TosaOpT>
+class ONNXAddSubOpLoweringToTOSA : public OpConversionPattern<ONNXOpT> {
```

Comment:
This looks good, but please consider to have a more general name like `ONNXBinaryElementwiseOpLoweringToTOSA` if more ops (e.g. Min, Max, ...) fall into this category.

Reply:
You're right @tungld - we're trying to keep somewhat aligned with the related TFL/torch-to-tosa conversion passes, but this is an example where a generalization is appropriate in ONNX that isn't for TFL.